indico/modules/events/surveys/templates/management/import_survey_questionnaire.html

```diff
@@ -0,0 +1,3 @@
+{% from 'forms/_form.html' import simple_form %}
+
+{{ simple_form(form, back=_('Cancel'), form_header_kwargs={'id': 'survey-question-form', 'action': request.relative_url}) }}
```

Comment:
You don't need a template for this, the `jsonify_form()` function is a wrapper for `simple_form()` to avoid creating templates containing nothing but a `simple_form` call